pkg/crd/gen.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
```

Comment:
@gshirazi I can't comment on it, but you checked in a binary `cmd/main`. Please remove that, other than that the change looks good to me.
/hold

Reply:
done.